### ISSUE TITLE
Display alternate futures via toggles.

### ISSUE THREAD

**Issue by pawn002** [LonnyGomes/covid-sankey-vizualization#97]
For example what does the data look like if the real numbers are 50 times what is reported.

**Comment by LonnyGomes** [comment]
@pawn002 interesting thought. Would this inflate universally or would it be assigner per country?

**Comment by pawn002** [comment]
I would say universally. The numbers for case counts continue to be funny, so alternate futures would be a way to show what it means of the numbers were off by 600% like it’s been stated in the news

**Comment by LonnyGomes** [comment]
yeah, there are a lot of funny numbers. You don't have to look any farther than the UK which doesn't seem to be reporting "recovered" cases. I think I see where you're coming from. Will have to think a bit more of how that could work. Increasing the numbers wouldn't necessarily change the death rate, but the totals. Is that what you were after?

**Comment by pawn002** [comment]
good you mention the death rate!

the death numbers shouldn't change but the percent will.
emphasizing the change would be very insightful for a sizable portion of the audience.
